web/components/ui/SocialLinks/SocialLinks.tsx

```diff
@@ -30,7 +30,8 @@ export const SocialLinks: FC<SocialLinksProps> = ({ links }) => {
           <a
             key={link.platform}
             href={link.url}
-            className={styles.link}
+            className={styles.linkBackground}
+						title={link.platform}
```

Comment:
Since this is a string it'll need to support translations. https://owncast.notion.site/48ba2900908940238f2cb3e1d32e2f85?p=17c69c26ef79804184bffbc138e999c8&pm=s&pvs=25#17d69c26ef7980b08fcbfd41eb5d95ec